LICENSE

```diff
@@ -349,6 +377,10 @@ Refer to the above license for the full text.
 This product bundles influxdb, which is available under an MIT license.
 @vendor/github.com/influxdata/influxdb/*
 ./vendor/github.com/influxdata/influxdb/LICENSE
+Refer to the above license for the full text.
+
+This product bundles influxdb, which is available under an MIT license.
+@vendor/github.com/influxdata/influxdb/*
```

Comment:
Same here. Even if this winds up a bit too long, having both licenses listed where it's clear they both apply is worth it, imo.

Reply:
Combined influxdb pointers in 80ccbf0b2b